### PR TITLE
refactor : remove redundant code

### PR DESCRIPTION
**Pull request by sudipkundu999**
In this PR, I'm removing the codes that have been moved to CL and also changing color variables accordingly.

### CODE REVIEW

File: pages/wishlist/wishlist.css [hunk 1]
```diff
@@ -12,7 +12,7 @@ main{
 }
 
 .card{
-    background-color: var(--BG-WHITE);
+    background-color: var(--white);
```
Comment (varun13169) [inline pages/wishlist/wishlist.css:15]: Naming simple color names is not a good way to go

File: pages/cart/cart.css [hunk 2]
```diff
@@ -36,14 +31,14 @@ main{
...
-    background-color: var(--yellow);
+    background-color: var(--yellow-bright);
     color: var(--black);
     border-radius: 0;
 }
 
 
 .card{
-    background-color: var(--BG-WHITE);
+    background-color: var(--white);
```
Comment (varun13169) [inline pages/cart/cart.css:41]: Naming simple color names is not a good way to go

**Review by srishtigautam21 — APPROVED** [review]

**Review by varun13169 — APPROVED** [review]
Added two comment, nothing major and can be addressed in follow up PRs

File: pages/cart/cart.css (hunk 2 above)
Comment (sudipkundu999) [reply pages/cart/cart.css:41]: Noted. I will change it during refactoring.

File: pages/wishlist/wishlist.css (hunk 1 above)
Comment (sudipkundu999) [reply pages/wishlist/wishlist.css:15]: Noted. I will change it during refactoring.

**Pull request merged**